third_party/move/move-model/src/lib.rs

```diff
@@ -111,7 +111,14 @@ pub fn run_model_builder_in_compiler_mode(
         },
         Flags::model_compilation()
             .set_skip_attribute_checks(skip_attribute_checks)
-            .set_keep_testing_functions(compile_test_code),
+            .set_keep_testing_functions(compile_test_code)
+            .set_flavor(
+                if language_version != LanguageVersion::V1 {
```

Comment:
Better if we can move this to a separate PR.

Reply:
I need it because the parser wasn't able to reflect yet over the language version. Now he can, over the flavor.